practice/e-commerce-website/src/components/ThreeDotsBtn/ThreeDotsBtn.tsx

```diff
@@ -0,0 +1,9 @@
+const ThreeDotsBtn = () => {
+  return (
+    <button>
+      <img src="images/components/ThreeDotsBtn/three-dots.png" alt="three dots button" />
```

Comment:
@ronlechiht please try out https://reactsvgicons.com/react-svg-icons-guide for your icon component. Using svg to can improve performance score